scrinium.csl

```diff
@@ -0,0 +1,794 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" page-range-format="chicago" demote-non-dropping-particle="sort-only">
```

Comment:
does the journal allow submissions in multiple languages? Otherwise, please set a default locale, likely `default-locale="en-US"` or `"en-GB"`